siwiflasher.py

```diff
@@ -373,20 +373,19 @@ def da_mem(self, address, size, ftype, file_count=1, fota=NACK):
             ASSERT(r == ACK, "DA_MEM ACK")
 
         r = struct.unpack(">BB", self.send(NONE, 2)) #filecount + ACK
-        ASSERT(r[0] == file_count, "File count does not match")
-        format_acks = 0
+        #ASSERT(r[0] == file_count, "File count does not match")
         for i in range(file_count):
-            format_acks += struct.unpack(">I", self.send(NONE, 4))[0] # Format Ack Count for each file
-        self.send(NONE, 1) # ACK
+            format_acks = struct.unpack(">I", self.send(NONE, 4))[0] # Format Ack Count for each file
```

Comment:
format_acks should be counted as the number is sent based on size of file being flashed.